README.md

```diff
@@ -70,7 +70,8 @@ Creates a factory that can be used to create new muxers.
 `options` is an optional `Object` that may have the following properties:
 
 - `maxMsgSize` - a number that defines how large mplex data messages can be in bytes, if messages are larger than this they will be sent as multiple messages (default: 1048576 - e.g. 1MB)
-- `maxStreamsPerConnection` - a number that defines how many streams are allowed per connection (default: 1024)
+- `maxIncomingStreamsPerConnection` - a number that defines how many incoming streams are allowed per connection (default: 1024)
+- `maxOutgoingStreamsPerConnection` - a number that defines how many outgoing streams are allowed per connection (default: 1024)
```

Comment:
Considering a muxer is a single connection, these names could probably be `maxIncomingStreams` and `maxOutgoingStreams`.
Also maybe align naming with `'inbound' | 'outbound'` from interface-connection. `maxInboundStreams` / `maxOutboundStreams`?

Reply:
Inbound/outbound sounds good.

Single connection - I was thinking something similar, but you pass these options to the factory constructor rather than to each muxer instance, is it still clear?

Reply:
I guess the description of the option mentions they are per-connection.